features/_your_results/your_results_calculating_different_salary_frequencies.feature

```diff
@@ -9,7 +9,7 @@ Feature: displaying different salary frequencies
     Scenario: Salary Per Month calculation
 
     # Given I fill in my details
-    # And my salary_frequency is 'per Year'
+    # And my salary_frequency is 'per Month'
     # When I reach the Results Page
     # Then I should see results broken down by month
```

Comment:
sure, but I dont think I put that in and left them there because I didnt know why they were put in originally.